eventually/vote/models.py

```diff
@@ -36,17 +41,31 @@ class Vote(models.Model):
     """
 
     MULTI_CHOICES = (
-        ('O', 'One'),
-        ('M', 'Multi'),
+        ("O", 'One'),
+        ("M", 'Multi'),
     )
 
+    event = models.ForeignKey(Event, on_delete=models.CASCADE, null=True)
     is_active = models.BooleanField(default=True)
     is_extended = models.BooleanField(default=True)
     title = models.TextField(max_length=100)
     vote_type = models.CharField(max_length=2, choices=MULTI_CHOICES)
     create_at = models.DateTimeField(auto_now_add=True, editable=False)
     update_at = models.DateTimeField(auto_now=True)
 
+    def __repr__(self):
+        """
+            This magic method is redefined to show all fields of the Vote object.
+        """
+        return '{} {} {} {} {} {} {} {}'.format(self.id,
+                                                self.event,
```

Comment:
you have event null=True so it can be empty.
use  self.event.id if self.event else None,

Reply:
sorry but it(`{} {} {} {} {} {} {} {}`) is not readable.